src/backend/env/mod.rs

```diff
@@ -1223,7 +1223,7 @@ impl State {
                 continue;
             };
             // All miner rewards are burned.
-            if user.miner {
+            if user.mode == Mode::Mining {
```

Comment:
In case of `Mode::Credits` we expect the rewards to be 0, right. If that doesn't hold for some reason (e.g. a bug) would it make sense to burn those rewards?

Reply:
Definitely, thanks Ulan!